packages/ods/src/components/clipboard/src/components/ods-clipboard/ods-clipboard.scss

```diff
@@ -7,22 +9,42 @@
 }
 
 .ods-clipboard {
+  &__container {
+    box-sizing: border-box;
+    display: inline-flex;
+    border: var(--ods-border-width-sm) solid var(--ods-color-form-element-border-default);
```

Comment:
actual clipboard does not have border (container should copy the input readonly style only)